frontend/pages/policies/ManagePoliciesPage/components/PolicyRunScriptModal/PolicyRunScriptModal.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { useCallback, useState } from "react";
+import { useQuery } from "react-query";
+import { omit } from "lodash";
+
+import { DEFAULT_USE_QUERY_OPTIONS } from "utilities/constants";
+
+import scriptsAPI, {
+  IListScriptsQueryKey,
+  IScriptsResponse,
+} from "services/entities/scripts";
+
+import { IPolicyStats } from "interfaces/policy";
+import { IScript } from "interfaces/script";
+
+// @ts-ignore
+import Dropdown from "components/forms/fields/Dropdown";
+import Checkbox from "components/forms/fields/Checkbox";
+import TooltipTruncatedText from "components/TooltipTruncatedText";
+import DataError from "components/DataError";
+import Spinner from "components/Spinner";
+import CustomLink from "components/CustomLink";
+import Button from "components/buttons/Button";
+import Modal from "components/Modal";
+
+// name avoids class name conflicts with hosts/details/HostDetailsPage/modals/RunScriptModal
+const baseClass = "policy-run-script-modal";
```

Comment:
Hm, thoughts on calling this and the software one:

```
AutomationSoftwareModal
AutomationScriptModal
```

Noting this because we aren't actually running or installing anything in these modals, so the ambiguity isn't the fault of the places where we _are_ doing those things.

Reply:
Good idea! Will do

Reply:
Resolving, will make these changes in follow-up PR per discussion